addOns/graphql/src/main/java/org/zaproxy/addon/graphql/UrlBuilder.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.graphql;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import org.apache.commons.httpclient.URI;
+import org.apache.commons.httpclient.URIException;
+
+public class UrlBuilder {
+
+    // Suppress default constructor for noninstantiability
```

Comment:
Not necessary, it's implied (the following assertion can be removed too).

Reply:
Sorry for not being clear, with "not necessary" was referring to the comment not the constructor itself. We should keep it (and add one to `MessageValidator`).